src/Symfony/Component/Process/Tests/ExecutableFinderTest.php

```diff
@@ -109,18 +109,38 @@ public function testFindWithOpenBaseDir()
             $this->markTestSkipped('Cannot test when open_basedir is set');
         }
 
-        $initialOpenBaseDir = ini_set('open_basedir', \dirname(\PHP_BINARY).\PATH_SEPARATOR.'/');
+        $initialOpenBaseDir = ini_set('open_basedir', \dirname(\PHP_BINARY).\PATH_SEPARATOR.sys_get_temp_dir().\PATH_SEPARATOR.getcwd());
 
         try {
             $finder = new ExecutableFinder();
             $result = $finder->find($this->getPhpBinaryName());
-
             $this->assertSamePath(\PHP_BINARY, $result);
         } finally {
             ini_set('open_basedir', $initialOpenBaseDir);
         }
     }
 
+    /**
+     * @runInSeparateProcess
+     */
+    public function testFindWithSubdirectoryOfOpenBaseDir()
+    {
+        if ('\\' === \DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('Cannot run test on windows');
```

Comment:
why couldn't this run on Windows ?

Reply:
I don't know. I based the new test on the existing open basedir test. Let's try if it works on AppVeyor.